src/frequenz/client/microgrid/_component_data.py

```diff
@@ -87,6 +87,24 @@ class MeterData(ComponentData):
     * `-` means supply into the grid.
     """
 
+    reactive_power: float
+    """The 3-phase reactive power, in Volt-Ampere Reactive (VAr).
+
+    Represented in the passive sign convention.
+
+    * `+` means consumption from the grid.
+    * `-` means supply into the grid.
+    """
```

Comment:
Active/passive sign convention does not apply to reactive power. So this needs to be changed (here and everywhere else).

This is what we do currently: https://github.com/frequenz-floss/frequenz-api-microgrid/blob/2332734e86dd8c1d3bf4ee601b93d52525654a6f/proto/frequenz/api/microgrid/microgrid.proto#L390-L393
.. and we plan to flip it in v0.17.0: https://github.com/frequenz-floss/frequenz-api-microgrid/issues/222

Reply:
Okay, this is definitely out of my expertise, so I guess I will just copy&paste what the API docs say. BTW, what is `ve` in those comments? I thought it was volt-electron but apparently that doesn't exist (I guess I was mixing it up with `eV`), and that it was a copy&paste error, but it seems it is neither, so I wonder what is it.

Reply:
> I guess I will just copy&paste what the API docs say.

Yeah, sounds good. We need to flip it here when we move this to API v0.17.0.

> BTW, what is ve in those comments?

`-ve` => negative, and `+ve` => positive :)
Probably I'll change it `negative` and `positive` given this confused you.

Reply:
Haha, yeah, totally unexpected, I never saw that convention before.

Reply:
There seems to be an inconsistency, one says positive is capacitive (leading) and the other say the opposite (positive is inductive (lagging)):

* https://github.com/frequenz-floss/frequenz-api-common/blob/08538ce91ebec107668628c14498fa0ae00253ad/proto/frequenz/api/common/metrics/electrical.proto#L108-L112
* https://github.com/frequenz-floss/frequenz-api-common/blob/08538ce91ebec107668628c14498fa0ae00253ad/proto/frequenz/api/common/metrics/electrical.proto#L161-L165

Or is this correct? If it is not, which one is correct?

While trying to see if the inconsistency was also present in the current version, I noticed that we *lost* all documentation about what each metric represents, the unit etc. Is this correct or am I missing anything.

As far as I see now key are using a key-value approach, where the key is the metric ID and the value is either a float or an aggregated value (with max/min/avg/etc). I guess the documentation about each metric should now be in the `enum Metric` values, but there is pretty much nothing there.

If this is correct, I think we should re-add the documentation for each metric, it is a very valuable piece of information we are losing otherwise. :scream: 

One downside about the current approach compared to the old one, is we now need to duplicate the documentation (or come up with a clever way to cross-reference some central documentation) for metrics that are basically the same, like active/reactive/apparent power for each phase.

Reply:
Judging from https://github.com/frequenz-floss/frequenz-api-microgrid/issues/222 I assume negative = inductive is correct. So I updated the PR with this in mind.

Reply:
>There seems to be an inconsistency, one says positive is capacitive (leading) and the other say the opposite (positive is inductive (lagging)):

Thanks for pointing out. It looks like a documentastion error.
https://github.com/frequenz-floss/frequenz-api-common/blob/08538ce91ebec107668628c14498fa0ae00253ad/proto/frequenz/api/common/metrics/electrical.proto#L108-L112
This is wrong^ . Inductive power is Lagging (current lags voltage).

Reply:
>I assume negative = inductive is correct.

According to our present standards, yes. Once https://github.com/frequenz-floss/frequenz-api-microgrid/issues/222 is addressed, it will need to be flipped.